uberfire-structure/uberfire-structure-backend/src/test/java/org/guvnor/structure/backend/pom/PomStructureEditorTest.java

```diff
@@ -33,26 +36,33 @@
 
 public class PomStructureEditorTest {
 
+    private final String POM = "pom.xml";
     private PomStructureEditor editor;
     private Path tmpRoot, tmp;
-    private final String POM = "pom.xml";
+    private DynamicDependencyTypeConfigurationMap configurationMap;
+    private String JPA_HIBERNATE_VERSION;
 
     @Before
     public void setUp() throws Exception {
         tmpRoot = Files.createTempDirectory("repo");
         tmp = TestUtil.createAndCopyToDirectory(tmpRoot,
                                                 "dummy",
                                                 "target/test-classes/dummy_empty_deps");
+        configurationMap = new DynamicDependencyTypeConfigurationMap();
+        DependencyType depType = new FakeJPATypeDependency();
+        configurationMap.addDependencies(depType,
+                                         depType.getDependencies());
+        JPA_HIBERNATE_VERSION = configurationMap.getMapping().get(depType.getType()).get(0).getVersion();
```

Comment:
why jpa here? if it's a `random` item.. please try not use the JPA to avoid confusion with real JPA dependency

Reply:
@porcelli Is a Fake dependency to test the read of the mapping. why Random ?